docs/examples/rx/05_read_storage/index.js

```diff
@@ -0,0 +1,48 @@
+// Import the API
+const { ApiRx } = require('@polkadot/api');
+// Import dependencies from RxJs
+const { combineLatest, of, zip } = require('rxjs');
+const { first, switchMap } = require('rxjs/operators');
+
+// Our address for Alice on the dev chain
+const Alice = '5GoKvZWG5ZPYL1WUovuHW3zJBWBP5eT8CbqjdRY4Q6iMaDtZ';
+
+function main () {
+  // Create our API with a default connection to the local node
+  new ApiRx().isReady
+    .pipe(
+      switchMap((api) =>
+        combineLatest(
+          of(api),
+          api.query.system.accountNonce(Alice).pipe(first()),
+          api.query.timestamp.blockPeriod().pipe(first()),
+          api.query.session.validators().pipe(first())
+        )
+      )
+    )
+    // Then we're subscribing to the emitted results
+    .subscribe(([api, accountNonce, blockPeriod, validators]) => {
```

Comment:
Subscribe inside subscribe.

Here i see 2 solutions (pseudo-code)
1. Split into 2 observables
```
const obs1 = ApiRx().isReady().pipe(switchMap(api => combineLatest(...)))
obs1.subscribe(() => { /* put the 1st console.logs here*/ })
obs1.pipe(
  filter(validators && validators.length >0)
  switchMap(()=>zip(...))
).subscribe(/* the other console.logs*/)
```

2. Use `tap`, so that we know you are doing side-effects
```
pipe(
  switchMap(()=>combineLatest(...)),
  tap(()=>{/*the 1st console.logs*/}),
  switchMap(()=>zip(...))
).subscribe(console.log)
```

Reply:
damned... I'm just seeing this now :| 
I got it working but I'm going to give the tap() version a try tomorrow morning